Lib/dis.py

```diff
@@ -327,12 +327,18 @@ def _get_instructions_bytes(code, varnames=None, names=None, constants=None,
                           arg, argval, argrepr,
                           offset, starts_line, is_jump_target)
 
-def disassemble(co, lasti=-1, *, file=None):
+def disassemble(co, lasti=-1, *, file=None, recursive=False):
     """Disassemble a code object."""
     cell_names = co.co_cellvars + co.co_freevars
     linestarts = dict(findlinestarts(co))
     _disassemble_bytes(co.co_code, lasti, co.co_varnames, co.co_names,
                        co.co_consts, cell_names, linestarts, file=file)
+    if recursive:
+        for x in co.co_consts:
+            if hasattr(x, 'co_code'):
+                print(file=file)
+                print("Disassembly of %r:" % (x,), file=file)
+                disassemble(x, file=file, recursive=True)
```

Comment:
This has the potential to get very noisy if disassembling a whole module, so I've posted a design question back on the original issue: https://bugs.python.org/issue11822#msg291718

I'd prefer to keep that discussion on the issue tracker, so I won't duplicate it here.